.github/workflows/reusable_build_docker.yaml

```diff
@@ -80,6 +80,8 @@ jobs:
         run: |
           cd ${{ github.workspace }}/docker/driver-loader/
           docker build -t docker.io/falcosecurity/falco-driver-loader-legacy:${{ inputs.arch }}-${{ inputs.tag }} \
+            --build-arg VERSION_BUCKET=deb${{ inputs.bucket_suffix }} \
```

Comment:
Why this was not required before #2718 ?

Reply:
/hold I found another mistake :facepalm:

Reply:
These are optional arguments, by default it was somehow reusing the old `0.35.1` image instead of building the legacy one from master.